### ISSUE TITLE
Filling up tmp with m3u downloads

### ISSUE THREAD

**Issue by dicer** [rettenbs/de_radio_amarok#2]
I just noticed that since enabling this script, I get 4 files in /tmp/kde-user/ named something like amarokGg6266.m3u.part. They just keep growing as long as amarok is running. 

I think some of the URLs are broken and keep downloading binary data?

Edit: By "keep growing" I mean that I stopped amarok when those files were 1.1GB each and my /tmp folder was full

**Comment by rettenbs** [comment]
Can you determine which radio stations are problematic?

**Comment by dicer** [comment]
Not really. This happens right after starting amarok (version 2.8.0 using KDE 4.14.34). Without any of the radio stations in the playlist.

**Comment by dicer** [comment]
I think the following might be the case: Amarok has a PlaylistFileLoaderJob loading remote playlists. Your script has lots of remote playlists. So amarok loads them all, but it seems 4 of them are not actually m3us but the stream itself already? Just a guess.
